packages/xo-server/src/api/token.mjs

```diff
@@ -17,6 +18,16 @@ create.params = {
     optional: true,
     type: 'string',
   },
+  client: {
+    description:
+      'client this authentication token belongs to, if a previous token exists, it will be updated and returned',
+    optional: true,
+    type: 'object',
+    properties: {
+      id: { description: 'unique identifier of this client', type: 'string' },
+      label: { description: 'human description of this client', optional: true, type: 'string' },
```

Comment:
Unused for now, I've decided to use the token description instead because it's handled in xo-web.

I think I should remove this for now but keep `client` an object to allow passing more client info in the future.